.gitlab/pkg_metrics.yml

```diff
@@ -51,17 +53,26 @@ send_pkg_size-a6:
   script:
     - source /root/.bashrc && conda activate ddpy3
     - mkdir -p /tmp/deb/agent
```

Comment:
Let's make the directory naming consistent - how about we call them all "{arch}-agent", e.g. `arm64-agent`, `amd64-agent` etc?